apps/daimo-mobile/src/view/screen/send/SendTransferScreen.tsx

```diff
@@ -251,6 +281,26 @@ function SendConfirm({
     nav.navigate("SendTab", { screen: "SendTransfer", params: { recipient } });
   };
 
+  const rpcFunc = getRpcFunc(daimoChainFromId(account!.homeChainId));
+  const rpcHook = getRpcHook(daimoChainFromId(account!.homeChainId));
+
+  const homeCoin = getHomeCoinByAddress(account.homeCoinAddress);
+  const numTokens = dollarsToAmount(money.dollars, homeCoin.decimals);
+
+  // If account's home coin is not the same as the desired send coin, retrieve Uniswap swap route.
+  let route = null;
+  if (homeCoin !== coin) {
+    const result = rpcHook.getUniswapRoute.useQuery({
+      fromToken: homeCoin.token as Address,
+      fromAmount: `${numTokens}` as BigIntStr,
+      fromAccount: toEAccount(account),
+      toToken: coin.token as Address,
+      toAddr: recipient.addr,
+    });
```

Comment:
can't do conditional hooks

also, `getUniswapRoute` is currently super slow. i have an idea how we could make it fast by reusing logic from AccountV2